Examples/Io/Performance/ActsExamples/Io/Performance/SeedingPerformanceWriter.cpp

```diff
@@ -95,10 +97,20 @@ ActsExamples::ProcessCode ActsExamples::SeedingPerformanceWriter::finalize() {
       "/ nMatchedParticles) = "
       << aveNDuplicatedSeeds);
 
+  auto write_float = [&](float f, const char* name) {
+    TVectorF v(1);
+    v[0] = f;
+    m_outputFile->WriteObject(&v, name);
+  };
+
   if (m_outputFile != nullptr) {
     m_outputFile->cd();
     m_effPlotTool.write(m_effPlotCache);
     m_duplicationPlotTool.write(m_duplicationPlotCache);
+    write_float(eff, "eff_seeds");
+    write_float(fakeRate, "fakerate_seeds");
+    write_float(duplicationRate, "duplicaterate_seeds");
```

Comment:
Maybe add underscores for readability?

Reply:
I can add underscores if you want, but I just copied it from the ckf performance writer. 
For consistency I think I would have to change there as well.

Reply:
shouldn't the function itself not be named `writeFloat` or do we have a different convention for lambdas?
@paulgessinger

Reply:
Ah, I meant in the string not in the function name ^^